addons/microsoft_calendar/utils/event_id_storage.py

```diff
@@ -0,0 +1,10 @@
+IDS_SEPARATOR = ':'
+
+def combine_ids(ms_id, ms_uid):
+    if not ms_id:
+        return False
+    return IDS_SEPARATOR.join([ms_id, ms_uid if ms_uid else ''])
```

Comment:
```suggestion
    return ms_id + IDS_SEPARATOR + ms_uid if ms_uid else ''
```
?

Reply:
I think the result will be different in case `ms_uid` is `False`.
In this first case, the result will be `ms_id:`
In the second case, the result will be the emty string.

Reply:
Seems to just be a question of precedence e.g. 

    ms_id + (IDS_SEPARATOR + ms_uid if ms_uid else '')

?

Reply:
Yes, why not, for me it's the same :)
So, I changed to `ms_id + IDS_SEPARATOR + (ms_uid if ms_uid else '')` as we always want the separator even if the uid is false ;-)